### PR TITLE
[css/json/html] move to l10n 

### PR DESCRIPTION
**Pull request by aeschli**
For #164438

### CODE REVIEW

File: extensions/html-language-features/server/package.json [hunk 1]
```diff
@@ -13,7 +13,6 @@
     "vscode-html-languageservice": "^5.0.2",
     "vscode-languageserver": "^8.1.0-next.1",
     "vscode-languageserver-textdocument": "^1.0.7",
-    "vscode-nls": "^5.2.0",
```
Comment (TylerLeonhardt) [inline extensions/html-language-features/server/package.json:16]: Just to make sure... removing this from the server was intentional, yes?

File: extensions/json-language-features/client/src/languageStatus.ts [hunk 2]
```diff
@@ -270,34 +266,24 @@ export function createLimitStatusItem(newItem: (limit: number) => Disposable) {
 }
 
 const openSettingsCommand = 'workbench.action.openSettings';
-const configureSettingsLabel = localize('status.button.configure', "Configure");
-
-export function createFoldingRangeLimitItem(documentSelector: string[], settingId: string, limit: number): Disposable {
```
Comment (TylerLeonhardt) [inline extensions/json-language-features/client/src/languageStatus.ts:275]: assuming this was dead code?

**Review by TylerLeonhardt — DISMISSED** [review]
LGTM I saw a couple of strings that weren't localized:

https://insiders.vscode.dev/github/microsoft/vscode/blob/bd5787cb5f263c96276ce054a37e1ce83c463e46/extensions/css-language-features/client/src/cssClient.ts#L154

https://insiders.vscode.dev/github/microsoft/vscode/blob/bd5787cb5f263c96276ce054a37e1ce83c463e46/extensions/css-language-features/client/src/cssClient.ts#L162

So it might be worth looking for the `show*Message` API instances and making sure they're all covered.

File: extensions/json-language-features/client/src/languageStatus.ts (hunk 2 above)
Comment (aeschli) [reply extensions/json-language-features/client/src/languageStatus.ts:275]: Yes

**Review by TylerLeonhardt — APPROVED** [review]

**Pull request merged**